homeassistant/components/homematicip_cloud/device.py

```diff
@@ -21,6 +21,7 @@
 ATTR_SABOTAGE = 'sabotage'
 ATTR_STATUS_UPDATE = 'status_update'
 ATTR_UNREACHABLE = 'unreachable'
+ATTR_GROUPMEMBERUNREADCHABLE = 'member device unreachable'
```

Comment:
Typo. Also all attributes need to be lowercase using lowercase.

Reply:
fixed